charts/fluent-bit/templates/_pod.tpl

```diff
@@ -1,5 +1,9 @@
 {{- define "fluent-bit.pod" -}}
-{{- if ne .Values.serviceAccount.automountServiceAccountToken nil }}
+{{- if ne .Values.automountServiceAccountToken nil }}
+automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
+{{- else if ne .Values.serviceAccount.automountToken nil }}
+automountServiceAccountToken: {{ .Values.serviceAccount.automountToken }}
+{{- else if ne .Values.serviceAccount.automountServiceAccountToken nil }}
```

Comment:
```suggestion
{{- if ne .Values.automountServiceAccountToken nil }}
automountServiceAccountToken: {{ .Values.automountServiceAccountToken }}
{{- else if ne .Values.serviceAccount.automountServiceAccountToken nil }}
```

The `serviceAccount.automountToken` isn't relevant here as it's explicitly focused on the service account config.